Menu/src/Option.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;

```

Comment:
It is best practice to have documentation comment.

Reply:
Please refer below for documentation comments:

https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
okay.
Thank you!!